templates/imgproxy/index.ts

```diff
@@ -0,0 +1,71 @@
+import { createTemplate, Services, randomString } from "~templates-utils";
+
+export default createTemplate({
+  name: "imgproxy",
+  meta: {
+    description: "imgproxy is the blazing fast and secure image processing tool.",
+    changeLog: [{ date: "2022-08-03", description: "first release" }],
+    links: [
+      { label: "Website", url: "https://imgproxy.net" },
+      {
+        label: "Documentation",
+        url: "https://docs.imgproxy.net",
+      },
+      { label: "Github", url: "https://github.com/imgproxy/imgproxy" },
+    ],
+    contributors: [
+      { name: "Ivan Ryan", url: "https://github.com/ivanonpc-22" },
+    ],
+  },
+  schema: {
+    type: "object",
+    required: ["projectName", "serviceName", "domain"],
+    properties: {
+      projectName: {
+        type: "string",
+        title: "Project Name",
+      },
+      serviceName: {
+        type: "string",
+        title: "Service Name",
+        default: "imgproxy",
+      },
+      domain: {
+        type: "string",
+        title: "Domain",
+      },
+    },
+  } as const,
+  generate({ 
+    projectName,
+    serviceName,
+    domain,
+    imgproxykey = randomString(18),
+    imgproxysalt = randomString(18),
+  }) {
+    const services: Services = [];
+
+    services.push({
+      type: "app",
+      data: {
+        projectName,
+        serviceName: serviceName,
+        env: [
+        `IMGPROXY_KEY=${imgproxykey}`,
+        `IMGPROXY_SALT=${imgproxysalt}`,
+        ].join("\n"),
```

Comment:
The formatting doesn't seem right. You should install prettier in your code editor and enable "format on save". Let me know if you need futher help with this.